mlflow/models/evaluation/utils/trace.py

```diff
@@ -123,6 +123,12 @@ def _setup_autolog(module):
                         AUTOLOGGING_INTEGRATIONS.pop(flavor, None)
                 except ImportError:
                     pass
+                except Exception as e:
+                    if original_config is None or not original_config.get("silent", False):
+                        _logger.warning(
+                            "Exception raised while calling autologging for %s: %s",
+                            flavor, str(e),
+                        )
```

Comment:
I think we don't need this, we should not raise exception in spark autolog if disable==True